src/llms-txt/infrastructure/markdown-services/content-types-collector.php

```diff
@@ -145,9 +156,55 @@ private function get_recent_cornerstone_content( string $post_type, int $limit )
 	 * @param string $post_type The post type.
 	 * @param int    $limit     The maximum number of posts to return.
 	 *
-	 * @return array<int, array<WP_Post>> The most recently modified posts.
+	 * @return array<WP_Post> The most recently modified posts.
 	 */
 	private function get_recent_posts( string $post_type, int $limit ): array {
+		$exclude_old = false;
+
+		if ( $post_type === 'post' ) {
+			$exclude_old = true;
+		}
+
+		if ( $this->indexable_helper->should_index_indexables() ) {
+			return $this->get_recently_modified_posts_indexables( $post_type, $limit, $exclude_old );
+		}
+
+		return $this->get_recently_modified_posts_wp_query( $post_type, $limit, $exclude_old );
+	}
+
+	/**
+	 * Returns most recently modified posts of a post type, using indexables.
+	 *
+	 * @param string $post_type   The post type.
+	 * @param int    $limit       The maximum number of posts to return.
+	 * @param bool   $exclude_old Whether to exclude posts older than one year.
+	 *
+	 * @return array<WP_Post> The most recently modified posts.
+	 */
+	private function get_recently_modified_posts_indexables( string $post_type, int $limit, bool $exclude_old ) {
```

Comment:
In a next iteration I would love us to split out the `old` functionality and not have a boolean determine behaviour. Either allow for extra filters to be passed or just have a _older_than_year esq method. But for now its fine.

Reply:
I'll create a tech debt task tomorrow about this.